chef_master/source/chef_license_accept.rst

```diff
@@ -103,18 +115,31 @@ This tool walks users through the install and setup of Automate.
 The Chef license is accepted after that in the browser.
 Please follow the in-product prompts.
 
+Chef Infra Server
+-----------------------------------------------------
+When installed as a system package users accept the license with the ctl command, EG ``chef-server-ctl reconfigure --chef-license=accept``.
+Acceptance can also be set in the configuration file ``chef-server.rb`` as ``chef_license "accept"``.
+
+Chef Infra Server is also distributed as a Habitat package and ran using the Habitat supervisor. In this mode users accept the license by setting the correct Habitat configuration values. The key is ``chef_license.acceptance``.
+
+For example: against a supervisor running Chef Infra Server run ``echo "chef_license.acceptance = accept" | hab config apply server.default 100``. See the Habitat `config updates <https://www.habitat.sh/docs/using-habitat/#config-updates>`__ documentation for more information about how to apply this configuration to a service group.
+
+Push Jobs Server
+-----------------------------------------------------
+TODO
```

Comment:
@markan Could I get some help with this? PJS is only installed into an existing Chef Server install, correct?